docs/tutorialkit.dev/astro.config.ts

```diff
@@ -19,7 +19,11 @@ export default defineConfig({
         light: './src/assets/brand/tutorialkit-logo-light.svg',
         replacesTitle: true,
       },
-      plugins: [starlightLinksValidator()],
+      plugins: [
+        starlightLinksValidator({
+          exclude: ['../guides/**/*'],
```

Comment:
I think this is related to https://github.com/HiDeoo/starlight-links-validator/issues/39.